docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md

```diff
@@ -0,0 +1,139 @@
+:::note
+   Before setting up the integration, test out the query with the checkpointing logic and a specific checkpoint value in the Google BigQuery console.
+:::
+To configure an Google BigQuery Source:
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
+1. On the Collection page, click **Add Source** next to a Hosted Collector.
+1. Search for and select **Google BigQuery**.<br/><img src={useBaseUrl('img/send-data/google-bigquery-icon.png')} alt="google-bigquery-icon" width="80" />
+1. Enter a **Name** for the Source. The description is optional. <br/><img src={useBaseUrl('img/send-data/google-bigquery_config_main.png')} alt="google-bigquery-config-main.png" width="400" />
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. **Project ID**. Enter the unique identifier number for your project. You can find this from the Google Cloud Console.
+1. **Delegated User Email**. Enter the admin user email address for the domain which grants the access to service account.
+1. **Checkpoint Field**. Enter the name of the field in the query result to be used for checkpointing. This field has to be increasing and of type number or timestamp.
+1. **Checkpoint Start**. Enter the first value for the checkpoint that the integration will plug into the query.
+1. **Time Field (Optional)**. Enter the name of the field in the query result to be parsed as timestamp. If not provided the current time will be used.
+1. **Query**. Enter the BigQuery Name of the user to run. It has to include the phrase "%CHECKPOINT%".
+1. **Query Interval (Optional)**. Enter the time interval to run the query in the format: Xm (for X minutes) or Xh (for X hours).
+1. **Google BigQuery Credential**. Upload the JSON Key file downloaded from Google Cloud IAM & Admin.
+1. **Processing Rules for Logs (Optional)**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+1. When you are finished configuring the Source, click **Submit**.
+
+### Error Types
```

Comment:
Missing the section on "On Query, Checkpoint and Checkpoint Start" - see README file in main branch.

Reply:
added